### ISSUE TITLE
Can't download dependencies

### ISSUE THREAD

**Issue by shawn-dsz** [yarnpkg/yarn#4586]
version: 1.1.0
Error: **error:0906D06C:PEM routines:PEM_read_bio:no start line**
running `> yarn --verbose` gives the following
```
... -- skipped for brevity ---
verbose 1.397 Performing "GET" request to "https://registry.yarnpkg.com/async/-/async-2.3.0.tgz".
verbose 1.405 Performing "GET" request to "https://registry.yarnpkg.com/body-parser/-/body-parser-1.17.1.tgz".
verbose 1.407 Performing "GET" request to "https://registry.yarnpkg.com/bytes/-/bytes-2.4.0.tgz".
verbose 1.424 Performing "GET" request to "https://registry.yarnpkg.com/content-type/-/content-type-1.0.4.tgz".
verbose 1.425 Performing "GET" request to "https://registry.yarnpkg.com/debug/-/debug-2.6.1.tgz".
verbose 1.434 Error: https://registry.yarnpkg.com/async/-/async-2.3.0.tgz: error:0906D06C:PEM routines:PEM_read_bio:no start line
    at Object.createSecureContext (_tls_common.js:85:17)
    at Object.exports.connect (_tls_wrap.js:1029:48)
    at Agent.createConnection (https.js:83:22)
    at Agent.createSocket (_http_agent.js:195:26)
    at Agent.addRequest (_http_agent.js:157:10)
    at new ClientRequest (_http_client.js:212:16)
    at Object.request (http.js:26:10)
    at Object.request (https.js:206:15)
    at Request.module.exports.Request.start (/Users/*user*/.yarn/lib/cli.js:122484:32)
    at Request.module.exports.Request.end (/Users/*user*/.yarn/lib/cli.js:123254:10)
error An unexpected error occurred: "https://registry.yarnpkg.com/async/-/async-2.3.0.tgz: error:0906D06C:PEM routines:PEM_read_bio:no start line".
info If you think this is a bug, please open a bug report with the information provided in "/Users/*user*/*proj*/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
verbose 1.446 Performing "GET" request to "https://registry.yarnpkg.com/raw-body/-/raw-body-2.2.0.tgz".
verbose 1.46 Performing "GET" request to "https://codeload.github.com/seek-oss/BuckyClient/tar.gz/c4c1d2e659872c03ae0ebc023dcdb3cea00f03c9".
verbose 1.463 Performing "GET" request to "https://registry.yarnpkg.com/xmlhttprequest/-/xmlhttprequest-1.6.0.tgz".
```

**Comment by BYK** [comment]
You are probably using a custom root certificate or there is something in your network intercepting TLS connections?

**Comment by shawn-dsz** [comment]
How can I fix this?

**Comment by BYK** [comment]
Closing due to lack of response. You can reopen when you are back with more info.

**Comment by shawn-dsz** [comment]
I don't think so, any ideas on how I can fix this.

**Comment by BYK** [comment]
>I don't think so, any ideas on how I can fix this.

Can you inspect your connection to that URL to see what is happening? Since you are the only one reporting this issue right now it is quite likely that there's something specific in your setup causing the issue. It is possible that there is a bug but we need to identify the root cause to be able to fix it first.

**Comment by brandonsturgeon** [comment]
Hi @BYK, I'm having this issue too. I just set all of the keys you mentioned in this comment: https://github.com/yarnpkg/yarn/issues/841#issuecomment-332868519

And now I'm getting this same error:
```
error An unexpected error occurred: "https://npm.my-company.site/package-name: error:0906D06C:PEM routines:PEM_read_bio:no start line".
```

**Comment by 3af** [comment]
Hi @brandonsturgeon,

I think you shouldn't set all the values together. I had the same error when I tried setting `cert` and `cafile` at the same time (maybe I just set incorrect values, I don't know).

I used `yarn config set cafile "C:\your\path\to\certificate\Root CA.cer"` and then it started working for me. It's important the file is in [PEM format](http://info.ssl.com/article.aspx?id=12149) (the extension doesn't matter).
